source/constructs/test/__snapshots__/live-streaming.test.ts.snap

```diff
@@ -1229,7 +1234,7 @@ Object {
         PolicyDocument: Object {
           Statement: Array [
             Object {
-              Action: *,
+              Action: s3:*,
```

Comment:
is this s3 permission necessary if there is a * permission above?

Reply:
This is part of the AWS Solutions Construct

Amazon S3 to CloudFront. They scoped the permissions down from * to s3:* 

https://www.npmjs.com/package/@aws-solutions-constructs/aws-cloudfront-s3